doc/release-notes/0.31/0.31.md

```diff
@@ -87,9 +87,40 @@ The v0.31.0 release contains the following known issues and limitations:
 <th valign="bottom">Impact</th>
 <th valign="bottom">Workaround</th>
 </tr>
-
 </thead>
+
 <tbody>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/13767">#13767</a></td>
+<td valign="top">Compressed references mode is not available</td>
+<td valign="top">AArch64 macOS (early access)</td>
```

Comment:
You should say "Apple M1 macOS" rather than "AArch64" to align with Apple's terminology.  Same with the other rows in this table.

Reply:
I guess "Apple Silicon macOS" is better.

Reply:
I changed the Platform column: "AArch64 macOS" -> "Apple Silicon macOS".